ga4gh/cli.py

```diff
@@ -401,10 +434,10 @@ def _textOutput(self, gaObjects):
             # TODO add in some more useful output here.
             print(read.id)
 
-
 # ListReferenceBases is an oddball, and doesn't fit either get or
 # search patterns.
 
+
```

Comment:
I installed via pip install -r requirements.txt, and used a virtualenv, so the version should be the same.

```
(server-ve)$:~/workspace/server$ flake8 --version
2.3.0 (pep8: 1.6.2, mccabe: 0.3.1, pyflakes: 1.0.0) CPython 2.7.9 on Linux
(server-ve)$:~/workspace/server$ cat requirements.txt | grep flake8
flake8==2.3.0  # Set due to conflict with pep8 in version 2.4.0
```